tests/lib/custom_lib/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2021, Legrand North America, LLC.
 # SPDX-License-Identifier: Apache-2.0
 
-cmake_minimum_required(VERSION 3.13.1)
+cmake_minimum_required(VERSION 3.20.0)
```

Comment:
I don't expect you are updating this line in every CMakeLists.txt file in the Zephyr ecosystem whenever Zephyr requires a higher version.  If this file does not require the higher version, then why change it here rather than leave it to be caught in `find_package(Zephyr)`